polaris-react/src/components/Columns/Columns.tsx

```diff
@@ -35,11 +45,11 @@ export interface ColumnsProps {
 
 export function Columns({children, columns, gap = '4'}: ColumnsProps) {
   const style = {
-    '--pc-columns-xs': formatColumns(columns?.xs || 6),
-    '--pc-columns-sm': formatColumns(columns?.sm),
-    '--pc-columns-md': formatColumns(columns?.md),
-    '--pc-columns-lg': formatColumns(columns?.lg),
-    '--pc-columns-xl': formatColumns(columns?.xl),
+    ...getResponsiveValue(
+      'columns',
+      'grid-template-columns',
+      formatColumns(columns || 6),
```

Comment:
I feel like we should just remove the default column value now. Can be a follow up and should probably be a quick conversation with the team